CONTRIBUTING.MD

```diff
@@ -0,0 +1,164 @@
+
+# Contributing to Kleros
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to Kleros and its packages, which are hosted in the [Kleros](https://github.com/kleros) on GitHub. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+#### Table Of Contents
+
+[Code of Conduct](#code-of-conduct)
+
+[How Can I Contribute?](#how-can-i-contribute)
+  * [Reporting Bugs](#reporting-bugs)
+  * [Suggesting Enhancements](#suggesting-enhancements)
+  * [Your First Code Contribution](#your-first-code-contribution)
+
+[Styleguides](#styleguides)
+  * [Git Commit Messages](#git-commit-messages)
+  * [JavaScript Styleguide](#javascript-styleguide)
+  * [Specs Styleguide](#specs-styleguide)
+  * [Documentation Styleguide](#documentation-styleguide)
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the [Kleros Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to [kleros@github.com](mailto:kleros@github.com).
```

Comment:
Is this email set?

Reply:
I tried but it's invalid. Let's remove the conduct report.

Reply:
Oops, yeah my bad.